source/main/ipc/electronStoreConversation.js

```diff
@@ -32,6 +32,9 @@ const reset = async () => {
   await unset(keys.CURRENCY_ACTIVE);
   await unset(keys.CURRENCY_SELECTED);
   await unset(keys.DATA_LAYER_MIGRATION_ACCEPTANCE);
+  await unset(keys.DISCREET_MODE_ENABLED);
+  await unset(keys.DISCREET_MODE_SETTINGS_TOOLTIP);
+  await unset(keys.DISCREET_MODE_NOTIFICATION);
```

Comment:
Did we add this to our "technical debt" list? My heart is bleeding because we still add stuff to pile of mud in our codebase :(

Reply:
We did not. @DominikGuzei would you like to create one describing the next steps for storage/electron bridge?